catalog.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <meta charset="UTF-8">
+  <title>Catalog</title>
+</head>
+<body>
+  <header>
+    <nav>
+      <a href="index.html">
+        <img src="https://via.placeholder.com/138x70" alt="Логотип" width="138" height="70">
+      </a>
+      <ul>
+        <li><a href="#">Информация</a></li>
+        <li><a href="#">Фото и видео</a></li>
+        <li><a href="#">Карта штата</a></li>
+        <li><a href="#">Гостиницы</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+   <section>
+    <h3>Найдено:3</h3>
+    <p>Сортировка:</p>
+      <ul>
+        <li><a href="#">По цене</a></li>
+        <li><a href="#">По типу</a></li>
+        <li><a href="#">По рейтингу</a></li>
+      </ul>
+
+      <ul>
+        <li>
+          <article>
+            <img src="https://via.placeholder.com/138x70" alt="Гостиница" width="138" height="70">
+            <h3>Amara Resort & Spa</h3>
```

Comment:
```suggestion
            <h3><a href="#!">Amara Resort & Spa</a></h3>
```

И так во всех карточках